config.md

```diff
@@ -61,7 +61,7 @@ This specification uses the following terms:
 Here is an example image JSON file:
```

Comment:
@wking @vbatts dropped the renaming https://github.com/opencontainers/image-spec/pull/231/files